### ISSUE TITLE
Feature Request: Option to sort by value/size instead of value

### ISSUE THREAD

**Issue by james-schwartzkopf** [BONNePlayground/VaultHuntersJewelSorting#2]
Since not every one cuts their jewels down to size 10, the import thing is often value per size, not the absolute value.

Could you add an option to sort by the attributes value / size?  

I've hacked that functionality into the SortingHelper and find it works very well locally when combine with Kendrome's Vault Hunter Tweaks (adds value / size to the tool tip for jewels).  I'd submit a PR, but my java is very rusty, and I couldn't even get the project to build properly.

Here is the code I was using:
```
            else if (
            		AttributeHelper.isFloatAttribute(leftAttribute)
            		|| AttributeHelper.isIntegerAttribute(leftAttribute)
            		|| AttributeHelper.isDoubleAttribute(leftAttribute)
            		)
            {
                int leftSize = leftData.getFirstValue(ModGearAttributes.JEWEL_SIZE).orElse(10);
                int rightSize = rightData.getFirstValue(ModGearAttributes.JEWEL_SIZE).orElse(10);

            	double leftValue = ((Optional<Number>) leftData.getFirstValue(leftAttribute)).orElse(0D).doubleValue();
            	double rightValue = ((Optional<Number>) rightData.getFirstValue(leftAttribute)).orElse(0D).doubleValue();
                
            	double leftValueSize = leftValue / leftSize;
            	double rightValueSize = rightValue / rightSize;
            	
            	returnValue = Double.compare(leftValueSize, rightValueSize);
                returnValue = ascending ? returnValue : returnValue * -1;
            }

```

**Comment by BONNe** [comment]
The issue with implementing this is that I do not know what is jimathy intention in implementing this mod into VHIntegrations.
If his integration will not contains this option I will add it.

**Comment by BONNe** [comment]
If you need help with custom build for now, I could help with that

**Comment by BONNe** [comment]
Implemented in 2.0.0
